Makefile

```diff
@@ -318,9 +318,18 @@ config_updater_vault_secret:
 		--context=$(cluster) \
 		--confirm \
 		--kubeconfig=/_kubeconfig
-	mkdir -p $(build_farm_credentials_folder)
-	oc --context "$(cluster)" sa create-kubeconfig -n ci config-updater > "$(build_farm_credentials_folder)/sa.config-updater.$(cluster).config"
-	make dry_run=false ci-secret-generator
+
+	./clusters/psi/create_kubeconfig.sh "$(build_farm_credentials_folder)/sa.config-updater.${cluster}.config" ${cluster} $@ ci ${API_SERVER_URL} config-updater-token-version-$(token_version)
+
+	echo "We have following secrets:"
+	ls $(build_farm_credentials_folder)
+
+	oc --context app.ci -n ci create secret generic config-updater \
+		--from-file=$(build_farm_credentials_folder) \
+		--dry-run=client -o json | oc --context app.ci apply --dry-run=${DRY_RUN} --as system:admin --server-side -f -
```

Comment:
`--server-side` is not really needed here but it seems the impact is only on one secret. So I am OK if you want it badly. 😉

Reply:
No its not needed, but for some reasons (I didn't investigate), the secret was created with server-side (no last-applied-configuration annotation) originally.